bedrock/base/geo.py

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+from django.conf import settings
+
+try:
+    import maxminddb
+except ImportError:
+    maxminddb = None
+
+
+logger = logging.getLogger(__name__)
```

Comment:
I'm a little concerned about the performance impact of the logging calls anyway, so if we don't have an immediate use for them I'd rather remove them.